addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -3034,6 +3047,23 @@ var BasicModel = AbstractModel.extend({
         var viewType = options && options.viewType || element.viewType;
         return Object.keys(fieldsInfo && fieldsInfo[viewType] || {});
     },
+    /**
+     * Get many2one fields names in a datapoint. This is useful in order to
+     * fetch their names in the case of a default_get.
+     *
+     * @private
+     * @param {Object} datapoint a valid resource object
+     * @returns {string[]} list of field names that are many2one
+     */
+    _getMany2OneFieldNames: function (datapoint) {
```

Comment:
shouldn't you call this function for the initial default_get as well? Otherwise, it is maybe no necessary...